src/en/howto-privatecloud.md

```diff
@@ -230,36 +255,47 @@ The output from the previous command will be similar to the following:
 The URL for the Object Store is listed against the `internalurl` field above
 and we refer to this as **$SWIFT_URL** in the following commands.
 
+!!! Note: you can verify the url before bootstrap with
+wget $SWIFT_URL/simplestreams/images/streams/v1/index.json
+
 Enter the following command to register the endpoint with the Simplestreams
 service:
 
 ```bash
-openstack endpoint create --region $REGION --publicurl $SWIFT_URL/simplestreams/images \
+openstack endpoint create --region $REGION \
+   --publicurl $SWIFT_URL/simplestreams/images \
```

Comment:
I don't think this is right. This bases the product-streams publicurl on the object-stores internal url ($SWIFT_URL contains the internal url). The public url should be based on the object-stores public url.

Reply:
using both public url and internal url now.